app/observers/manual_observers_registry.rb

```diff
@@ -19,6 +23,14 @@ def creation
     []
   end
 
+  def withdrawal
+    [
+      publishing_api_withdrawer,
+      rummager_withdrawer,
+      panopticon_exporter,
+    ]
```

Comment:
Logging this would be a good thing to do, but will require changing `PublicationLog` so it can distinguish between publication and withdrawal events. That would be a good time to revisit the responsibilities of that class – it's currently doubling up as an event log and a way to build change history.

If we want to tackle it, I think it would be best done as part of the story but not as part of this PR. The change here provides a useful improvement by itself, as it automates a lot of what 2nd line have to do by hand when withdrawing a manual, so I think we should merge as is.